next.config.js

```diff
@@ -1,9 +1,6 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
-  experimental: {
-    missingSuspenseWithCSRBailout: false,
-  },
```

Comment:
Stood-out, just noting in case it was unintentional.

Reply:
yep. this is no longer needed and actually throws a warning during the build.